flake.nix

```diff
@@ -302,8 +330,10 @@
           };
 
           gitlab-ci = {
-            baseUrl = "https://gitlab.horizon-haskell.net/nix/gitlab-ci";
-            flakeRef = "git+https://gitlab.horizon-haskell.net/nix/gitlab-ci";
+            imports = [ gitlab ];
+            forgeRoot = "https://gitlab.horizon-haskell.net/";
+            owner = "nix";
+            repo = "gitlab-ci";
```

Comment:
Against odds, the only GitLab input we have is a self-hosted one. So I commented out the `forgeRoot` and `gitlab.com` seems to render well.